### PR TITLE
Update getting started material MODCPCT-30

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: doc/getting-started.md [hunk 1]
```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
...
+yaz4j is distributed in https://maven.indexdata.com APT repo.
+If you are using Ubuntu, it is not necessary to compile and install yaz4j.
+
+Steps for install yaz4j (not necessary on Ubuntu amd64).
+In the directory next to mod-copycat checkout, install as follows:
 
 	term1$ sudo apt install yaz libyaz-dev swig
 	term1$ git clone https://github.com/indexdata/yaz4j.git
 	term1$ cd yaz4j
 	term1$ git checkout v1.6.0
```
Comment (MikeTaylor) [inline doc/getting-started.md:34]: Please remind me why we invite people to lock to v1.6.0?

File: doc/getting-started.md [hunk 2]
```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
...
 Once the yaz4j library is available, mod-copycat itself can be built:
 
-	term1$ cd ../mod-copycat
+	term1$ cd mod-copycat
 	term1$ mvn install
 
 ## To run
 
-	term1$ java -jar target/mod-copycat-fat.jar
+You must have PostgresQL running. Set environment appropriately:
```
Comment (MikeTaylor) [inline doc/getting-started.md:53]: The canonical capitalization is "PostgreSQL".

File: doc/getting-started.md [hunk 3]
```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
...
+	$ java -jar target/mod-copycat-fat.jar
 
 ## To initialize
 
 In another terminal:
 
-	term2$ curl -d'{"module_to":"foo"}' -HX-Okapi-Tenant:testlib "-HAccept:*/*" -HContent-Type:application/json http://localhost:8081/_/tenant
-	[]
-	term2$ 
+	$ curl -d'{"module_to":"foo","parameters":[{"key":"loadReference","value":"true"}]}' \
```
Comment (MikeTaylor) [inline doc/getting-started.md:69]: What does the "foo" mean here? I don't see any mention of mod-copycat in the invocation, so how does Okapi know what module to initialize? Could it be that we meant to say "mod-copycat" and accidentally said "foo" instead?

File: doc/getting-started.md [hunk 4]
```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
...
-In order to complete this operation mod-copycat must be able to contact Okapi and in turn mod-source-record-manager to complete the operation.
-
-## Integrating into a FOLIO-backend Vagrant box
-
-Create the configuration file for a FOLIO core backend VM:
-
-	$ vagrant init folio/snapshot-backend-core
-
-Now start the VM for the first time. This will take some time, as it will need to download the VM image:
+We have two profiles part of reference data, one for OCLC Worldcat and
```
Comment (MikeTaylor) [inline doc/getting-started.md:108]: Maybe "two profiles AS part of"?

File: doc/getting-started.md [hunk 5]
```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
...
+    $ okapi-curl /users?query=username%3D%3Duser | jq .users[0].id
 
-	yarn stripes mod update --okapi http://localhost:9130
-	yarn stripes mod disable --okapi http://localhost:9130 --tenant diku
-	yarn stripes mod enable --okapi http://localhost:9130 --tenant diku
-	echo ui-inventory.settings.single-record-import | yarn stripes perm assign --user diku_admin --okapi http://localhost:9130 --tenant diku
+Finally, we're ready to make an import. Replace userid - no quotes. We
+import LC number `2004436018 `- title `Ole Lukøie`.
 
+	$ okapi-curl /copycat/imports -HX-Okapi-User-Id:userid \
```
Comment (MikeTaylor) [inline doc/getting-started.md:143]: Why do we need to specify `X-Okapi-User-Id` — isn't that implied by the token?

File: doc/getting-started.md [hunk 6]
```diff
@@ -9,133 +9,142 @@ mike@indexdata.com
...
+        -d'{"externalIdentifier":"2004436018","profileId":"8594713d-4525-4cc7-b138-a07db4692c37"}'
+    profileId":"8594713d-4525-4cc7-b138-a07db4692c37"}'
+    {
+      "externalIdentifier" : "2004436018",
+      "internalIdentifier" : "fa6856d2-83cb-41fb-a5fa-0a32505c2cb4",
+      "profileId" : "8594713d-4525-4cc7-b138-a07db4692c37"
+    }
```
Comment (MikeTaylor) [inline doc/getting-started.md:150]: Perhaps add a note at the end: "This reports that the the LCDB record with LCCN 2004436018 has been created as a FOLIO instance with UUID fa6856d2-83cb-41fb-a5fa-0a32505c2cb4."

**Review by MikeTaylor — APPROVED** [review]
Looks good!

File: doc/getting-started.md (hunk 1 above)
Comment (adamdickmeiss) [reply doc/getting-started.md:34]: Latest release. and it's easier to with with a fixed version.

File: doc/getting-started.md (hunk 3 above)
Comment (adamdickmeiss) [reply doc/getting-started.md:69]: Okapi is not involved here. And the module doesn't use this.

File: doc/getting-started.md (hunk 4 above)
Comment (adamdickmeiss) [reply doc/getting-started.md:108]: yes

File: doc/getting-started.md (hunk 5 above)
Comment (adamdickmeiss) [reply doc/getting-started.md:143]: SRS wants this. But I just realized that mod-copycat could start figuring it out for us. Yet another thing that's slightly sub optimal.

File: doc/getting-started.md (hunk 6 above)
Comment (adamdickmeiss) [reply doc/getting-started.md:150]: Yep. Good idea.

File: doc/getting-started.md (hunk 1 above)
Comment (MikeTaylor) [reply doc/getting-started.md:34]: Eaiser to ... I guess, write the `pom.xml` or something.

File: doc/getting-started.md (hunk 3 above)
Comment (MikeTaylor) [reply doc/getting-started.md:69]: Oh, stupid of me to have assumed Okapi!

**Pull request merged**